e2echart/e2e-chart-template.html

```diff
@@ -97,13 +97,50 @@ <h5 class="modal-title">Resource</h5>
         return false
     }
 
+    // If this is a leader found, lost, elected, or missing, return true and the reason.
+    // Otherwise, return false.
+    function isEtcdLeaderFoundOrLost(eventInterval) {
+        const { tempStructuredLocator, tempStructuredMessage } = eventInterval;
+        let reason = null;
+        if (
+            tempStructuredLocator &&
+            tempStructuredLocator.keys &&
+            tempStructuredLocator.keys.hasOwnProperty('namespace') &&
+            tempStructuredLocator.keys['namespace'].includes('openshift-etcd')
```

Comment:
This seems excessive, were all these checks required? tempStructuredLocator should always be there, as should keys, but it could be null. We want to be as concise as possible here so this doesn't spread, this file would be a mess if we do this every time we check a property.

Reply:
understood.  Since you can guarantee those fields will be present, removing.